pages/01.administrate/07.specific_use_cases/05.external_storage/external_storage.md

```diff
@@ -5,122 +5,248 @@ taxonomy:
 
-You should also have your external drive (plugged via USB or SATA).
+## [fa=list-alt /] Prerequisites
 
-## 1. Connect and identify the disk
+* Have some time at a moment when your server users can accept a shutdown. The steps to be performed, even if they are relatively simple, can sometimes seem technical and require in any case **to take your time**.
 
-Start by connecting your drive to the system. You shall then identify which name is used by the system to refer to the disk.
+* Know how to connect as root on your system, for example via [SSH](/ssh). (Note: while logged in as `admin`, you can root with `sudo su`)
 
-You can do this with this command :
+* Know the basic commands `cd`, `ls`, `mkdir`, `rm`.
+
+* Have a backup in case things don't work out as planned
+
+* Have extra storage (SSD, hard drive, USB stick) connected to your server via USB or SATA
+
+## 1. Identify directories to be moved
+
+The `ncdu /` command allows you to browse the folders on your server to see how big they are. 
+
+Below is an explanation of some of the paths that can take up weight with some comments to help you reduce their weight or choose to move them.
+
+| Paths | Contents | Tips |
+|--------|---|---|
+| /home | User folders accessible via SFTP | Moveable to a hard disk |
+| /home/yunohost.backup       | YunoHost's backups  | Depending on your backup strategy, you may want to place this folder on a separate drive from your data or databases.
+| /home/yunohost.app          |Heavy data from yunohost applications (nextcloud, matrix...)|Moveable to a hard disk
+| /home/yunohost.multimedia | Heavy data shared between several applications | Moveable to a hard disk |
+| /var/lib/mysql | Database used by applications | Ideally leave on SSD for performance reasons |
+| /var/lib/postgresql | Database used by applications | Ideally leave on SSD for performance reasons | /var/mail
+| /var/mail | User e-mails | Movable to a hard disk |
+/var/www | Program of installed web applications | Ideally leave on SSD for performance reasons |
+| /var/log | Event logs (pages consulted, connection attempts, hardware errors...). | This directory should not take up too much space, if it grows quickly, it may be a looping error that should be resolved.
+| /opt | Program and dependency of some YunoHost applications. | Ideally leave it on the SSD for performance reasons. For nodejs applications it is possible to do some cleanup of unused versions.
+| /boot | Kernels and boot files | Do not move unless you know what you are doing. It can happen that too many kernels are kept, it is possible to do some cleanup.
```

Comment:
```suggestion
| Paths | Contents | Tips |
|--------|---|---|
| `/home` | User folders accessible via SFTP | Moveable to a hard disk |
| `/home/yunohost.backup`       | YunoHost's backups  | Depending on your backup strategy, you may want to place this folder on a separate drive from your data or databases.
| `/home/yunohost.app`          |Heavy data from yunohost applications (nextcloud, matrix...)|Moveable to a hard disk
| `/home/yunohost.multimedia` | Heavy data shared between several applications | Moveable to a hard disk |
| `/var/lib/mysql` | Database used by applications | Ideally leave on SSD for performance reasons |
| `/var/lib/postgresql` | Database used by applications | Ideally leave on SSD for performance reasons | /var/mail
| `/var/mail` | User e-mails | Movable to a hard disk |
| `/var/www` | Program of installed web applications | Ideally leave on SSD for performance reasons |
| `/var/log` | Event logs (pages consulted, connection attempts, hardware errors...). | This directory should not take up too much space, if it grows quickly, it may be a looping error that should be resolved.
| `/opt` | Program and dependency of some YunoHost applications. | Ideally leave it on the SSD for performance reasons. For nodejs applications it is possible to do some cleanup of unused versions.
| `/boot` | Kernels and boot files | Do not move unless you know what you are doing. It can happen that too many kernels are kept, it is possible to do some cleanup.
```